epicbox-hyperskill/cpp/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.18.4
+
+RUN apk add --no-cache g++ make cmake
+
+RUN apk add --no-cache python3 openjdk17-jdk curl bash
+
+RUN apk add --no-cache python3-dev py3-pip gcc musl-dev linux-headers && \
```

Comment:
Need to do like epicbox-gcc
https://github.com/hyperskill/epicbox-images/blob/7b336aec23dc795ec08d758801d44f8ea3140572/epicbox-gcc/Dockerfile
```
...
ENV GCC_VERSION 10.2.1-1

RUN apt-get update && apt-get install -y --no-install-recommends \
    gcc=4:${GCC_VERSION} gcc-multilib=4:${GCC_VERSION} g++=4:${GCC_VERSION}
    ...
```